lk2nd/hw/regulator/gpl/qcom_spmi-regulator.c

```diff
@@ -2586,6 +2619,7 @@ static const struct spmi_regulator_match qcom_spmi_regulator_match[] = {
 	{ .subtype = PM8226_SUBTYPE,  .sid = 1, .data = pm8226_regulators },
 	{ .subtype = PM8841_SUBTYPE,  .sid = 5, .data = pm8841_regulators },
 	{ .subtype = PM8916_SUBTYPE,  .sid = 1, .data = pm8916_regulators },
+	{ .subtype = PM8937_SUBTYPE,  .sid = 1, .data = pm8937_regulators },
```

Comment:
This is meant to be synced completely with upstream to add all the new entries. I just did that and pushed to `main`, can you rebase to drop these changes?